fixham/internal/docs/dev.go

```diff
@@ -0,0 +1,103 @@
+package docs
```

Comment:
I think docs is an unintuitive name for this package - it seems to actually be responsible for publishing and interacting with the hashrelease artifacts (or at least some of them).

I think this might be better named as "publish" and then we can move all of the publish logic here?

Alternatively, this can all just be moved into the "hashrelease" package since the logic to publish and delete hash releases is very specific to hashreleases.

The release notes generation probably belongs in its own releasenote package?